bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+            Slider item=HttpOnlyAudioThreshold          
+            Default item=HttpOnlyAudioAlarm
+            Text label="Mjpeg Stream" icon="camera"{Video url="http://192.168.0.2:54323/ipcamera.mjpeg" encoding="mjpeg"}
+            Text label="snapshots 1FPS Stream" icon="camera"{Video url="http://192.168.0.2:54323/snapshots.mjpeg" encoding="mjpeg"}
+            Text label="autofps Stream" icon="camera"{Video url="http://192.168.0.2:54323/autofps.mjpeg" encoding="mjpeg"}
+            Text label="HLS Video Stream" icon="camera"{Video url="http://192.168.0.2:54323/ipcamera.m3u8" encoding="hls"}
+            Text label="HLS Stream" icon="camera"{Webview url="http://192.168.0.2:54323/ipcamera.m3u8" height=15}
+            Text label="Image jpg method" icon="camera"{Image url="http://192.168.0.2:54323/ipcamera.jpg" refresh=1000}                
+        }
+
+```
+
+
+
+*.rules
+
+```java
+rule "Move cameras direction"
+when
+    Item BabyCamDirection changed
+then
+    switch (BabyCamDirection.state as DecimalType) {
+        case 0 :{
+        //Room
+        BabyCamPan.sendCommand(22)
+        BabyCamTilt.sendCommand(60)
+        BabyCamZoom.sendCommand(0)
+        }
+        case 1 :{
+        //Cot
+        BabyCamPan.sendCommand(22)
+        BabyCamTilt.sendCommand(0)
+        BabyCamZoom.sendCommand(0)
+        }
+        case 2 : {
+        //Door
+        BabyCamPan.sendCommand(15)
+        BabyCamTilt.sendCommand(75)
+        BabyCamZoom.sendCommand(1)
+        }
+    }
+end
+
+rule "Camera detected crying"
+when
+    Item BabyCamAudioAlarm changed from OFF to ON
+then
+if(BabyMonitor.state==ON){
+
+    if(MumAlerts.state==ON){
+    sendNotification("mum@parentCo.com", "Mum, the baby is awake.")
+    }
+
+    if(DadAlerts.state==ON){
+    sendNotification("dad@parentCo.com", "Dad, the baby is awake.")
+    }
+
+    if(TvAlerts.state==ON){
+    myKodi_notification.sendCommand("Baby is crying.")
+    }
+}
+end
+
+rule "Create time of last movement"
+    when
+    Item BabyCamLastMotionType received update
+    then
+    BabyCamLastMotionTime.postUpdate( new DateTimeType() )
+end
+
+```
+
+Note: For the above notifications to work you will need to setup multiple users with the correct email address's at the openHAB cloud.
+
+
+## Moving PTZ capable cameras
+
+To move a camera with this binding you need an ONVIF camera that supports one of the following:
+
++ Absolute movements
++ Relative movements
++ Continuous movements  
++ Presets
+
+
+To test your cameras compatibility and also to create some preset locations, use a free program called ``onvif device manager`` (ODM for short).
+Not all Onvif cameras work with all of the methods, so testing first to confirm what works is a good idea and the presets can not be created with the binding, only loaded after they are already created.
+After creating new or changing existing presets, it may be necessary to restart the binding before they can be used. 
```

Comment:
The camera creates TOKENS which change when the presets change, sadly it is not as simple as saying 'load preset 1' the binding needs to request the list and then parse and store the tokens and I went with this option over polling the camera to check for any tokens that have changed. This could be improved but I don't see another way other than polling or perhaps using a REFRESH command on the channel to trigger an update.

Reply:
I made some changes to make things better is this area. A refresh command will fetch tokens without needing any restarts. Going to update the readme in the next day or two.